### DISCUSSION TITLE
Add images to a post

### DISCUSSION THREAD

**Discussion by MetaApo13** [Docile-Alligator/Infinity-For-Reddit#1220] [Duplicate Ideas] [answered]
A feature about Boost for Reddit that I really liked was the option to add images to a text post.

There would be a button for this next to bold & & italics. Selecting the image would automatically upload it to imgur & generate the corresponding link.

I find this useful for making posts which are mostly text with a few images for illustration.

**Comment by Wladefant** [comment, marked as answer]
Duplicate of #1106
